src/BookReader.js

```diff
@@ -473,6 +475,12 @@ BookReader.prototype.getInitialMode = function(params) {
   if (!this.canSwitchToMode(nextMode)) {
     nextMode = this.constMode1up;
   }
+
+  // override defaults mode via `bookreader-defaults` metadata
+  if (null != this.options.defaults) {
```

Comment:
this conditional is "[yoda style](https://eslint.org/docs/latest/rules/yoda)". we don't usually use this style in BR.

can a falsey work? `if (!this.options.defaults)` ?